loggo/loggo.py

```diff
@@ -482,7 +464,7 @@ def _force_string_and_truncate(self, obj: Any, truncate: Optional[int], use_repr
         try:
             obj = str(obj) if not use_repr else repr(obj)
         except Exception as error:
-            self.warning('Object could not be cast to string', extra=dict(error_type=type(error), error=error))
+            self.warning('Object could not be cast to string', extra=dict(exception_type=type(error), error=error))
```

Comment:
isnt the key name change here a bit inconsistent, since `exception_type` is the type of `error`

Reply:
the user can address exception_type as a formatter, and i think it's good to be clear about the difference between error and exception, as we discussed. maybe we do `except Exception as exc`, even though that's slightly unconventional